app/models/order_import.rb

```diff
@@ -110,15 +110,21 @@ def handle_save_clean_orders # TODO refactor and rename
 
   def handle_save_nothing_on_error # TODO refactor and rename
     Order.transaction do
-      CSV.open(upload_file_path, headers: true, skip_lines: /^,*$/).each do |row|
-        row_importer = import_row(row)
-        self.error_report += row_importer.row_with_errors.to_csv
-
-        if row_importer.errors?
-          result.failures += 1
-        else
-          result.successes += 1
+      begin
+        CSV.open(upload_file_path, headers: true, skip_lines: /^,*$/).each do |row|
+          row_importer = import_row(row)
+          self.error_report += row_importer.row_with_errors.to_csv
+
+          if row_importer.errors?
+            result.failures += 1
+          else
+            result.successes += 1
+          end
         end
+      rescue Exception => e
```

Comment:
We should probably just `rescue => e`. That'll just catch `StandardError`. It's usually not a good idea to `rescue Exception` as `Exception` includes a lot of things like interrupts.